pages/colors_symbols.md

```diff
@@ -5,11 +5,11 @@ edited: 26 November 2015
 
 Modifying map colors and symbols
 
-[Color Window](color_dock_widget.md){: .subpage} <br />
+[Color Window](color_dock_widget.md){: .subpage}  
```

Comment:
I'm sure the `<br />` is there intentionally, for the Markdown converter in doxygen which builds the manual.

Reply:
Something that needs to be tested(therefor my final paragraph). 

Usually Markdown parsers writes a `<br>` tag when there is two white spaces followed by a actual line break. Not used very often/not known about so assumed they should be replaced.

Reply:
As far as I can tell from the Doxygen docs the syntax should be supported.